src/test/scala/mesosphere/marathon/core/event/impl/stream/HttpEventSSEHandleTest.scala

```diff
@@ -0,0 +1,59 @@
+package mesosphere.marathon.core.event.impl.stream
+
+import javax.servlet.http.HttpServletRequest
+
+import mesosphere.marathon.test.{ MarathonSpec, Mockito }
+import org.eclipse.jetty.servlets.EventSource.Emitter
+import org.scalatest.{ GivenWhenThen, Matchers }
+import collection.JavaConversions._
+
+class HttpEventSSEHandleTest extends MarathonSpec with Matchers with Mockito with GivenWhenThen {
```

Comment:
Please use either `UnitTest` or `FunTest` as the base type. They have all of the other traits mixed in.